allennlp/common/tqdm.py

```diff
@@ -0,0 +1,22 @@
+"""
+:class:`~allennlp.common.tqdm.Tqdm` wraps tqdm so we can add configurable
+global defaults for certain tqdm parameters.
+"""
+
+from tqdm import tqdm as _tqdm
+
+class Tqdm:
+    default_mininterval: float = 0.1
```

Comment:
Maybe give a comment here that this is the default value in `tqdm`, so that this isn't just a magic number.